src/rgw/rgw_lc.h

```diff
@@ -502,6 +501,13 @@ class RGWLC : public DoutPrefixProvider {
   int handle_multipart_expiration(RGWRados::Bucket *target, const map<string, lc_op>& prefix_map);
 };
 
+using LCCheckExpiresResult = std::tuple<bool, std::string, std::string>;
```

Comment:
@mattbenjamin is this being used anywhere? If can be removed also remove the include above.

Reply:
ah, yes, that was something I abandoned and forgot to delete

Reply:
deleted